server/ui/ui/src/pages/namespace/Namespace.tsx

```diff
@@ -212,11 +213,18 @@ const Namespace: FC<Props> = ({ setSyncValue }:Props) => {
             </p>
             <div
               className="Namespace__Delete--buttons relative"
+              data-tip="Users cannot delete a namespace."
```

Comment:
```suggestion
              data-tip="Only administrators can delete a namespace."
```

Actually this should instead block anyone but an admin.